externs/olx.js

```diff
@@ -101,6 +101,11 @@ olx.LogoOptions.prototype.src;
  * @typedef {{map: (ol.Map|undefined),
  *     maxLines: (number|undefined),
  *     strokeStyle: (ol.style.Stroke|undefined),
+ *     showLabels: (boolean|undefined),
+ *     lonLabelFormatter: (Function|undefined),
+ *     latLabelFormatter: (Function|undefined),
+ *     lonLabelPosition: (number|undefined),
+ *     latLabelPosition: (number|undefined),
```

Comment:
You should add documentation for these options below.
Also the formatter functions can be more strongly typed (`function(number):string`), right?